tests/functional/spec/features/section_summary/section_summary.spec.js

```diff
@@ -84,6 +84,15 @@ describe("Section Summary", () => {
       expect(browser.getUrl()).to.contain(AddressDurationPage.pageName);
     });
 
+    it("When I select edit from Final Summary and change an answer and then go to the next question and click previous, Then I should return to the question I originally edited", () => {
```

Comment:
This wasn't the test I was expecting, I thought it would be the same as the summary one, i.e you go back from the section summary then click previous to get back to the section summary. Am I missing something, or do we have that already?

Reply:
This was an additional test in order to cover the scenario Mebin commented on here: https://github.com/ONSdigital/eq-questionnaire-runner/pull/738#discussion_r770393792

Reply:
Perhaps the title needs to be updated to make it clearer, but difficult to come up with a good given when, then

Reply:
Do we have one that covers a return to the section summary by not changing any answers and using previous?

Reply:
Not at the moment, I will add one, it is only covered in the unit test

Reply:
Added another test now 👍